i18n/ja/docusaurus-plugin-content-docs/current/pricing.md

```diff
@@ -2,76 +2,70 @@
+<summary>Momento を無料で本番環境のアプリケーションに使うことはできますか？</summary>
+もちろんです！無料枠や低い利用率でも、大規模利用のものと同様に、高速で可用性がありセキュアです。複数AZ レプリケーション、ホットキー保護、そして突発的なリクエストへの自動スケーリングといった高可用性のための機能が全て利用可能です。セキュリティのための全ての機能(エンドツーエンド暗号化、リクエスト毎の認証、TLS)も無料で使えます。
 
-We frequently find customers provisioning an entire cluster to handle low-RPS workflows. If you want them to be HA, you need multiple nodes. If you believe in CICD, you likely have a similarly sized cluster in staging—and maybe even in dev. That adds up! Turn off those machines—and save the environment. Go Momento!
+お客さんの中でよく、低いRPS のワークフローのためにフル装備のクラスターをプロビジョンしているケースを見かけます。HA (高可用性)が欲しい場合、複数ノードが必要です。CICD を利用したければ、同程度のサイズのクラスターを、ステージングや開発環境にさえ設置したくなるでしょう。全てコストとして積み上がります！こうしたマシンは停止してしまって、コストを削減しましょう。Momento で行きましょう！
 </details>
 
 <details>
-  <summary>Can I run a production app for $1.50/month on Momento?</summary>
-Absolutely! If you are transferring 60 GB of data in/out of Momento each month, you get the first 50GB free each month and pay $0.15/GB for the remaining 10GB each month. Access enterprise-grade availability, security, and performance at any scale with Momento.
+  <summary>Momento を毎月$1.50 で本番環境のアプリケーションに使うことはできますか？</summary>
+もちろんです！もし毎月60 GB のデータ転送の出入りがMomento にあったとすると、毎月最初の50GB 分は無料で、残りの毎月10GB に対して$0.15/GB を支払うことになります。どんなスケールでも、Momento のエンタープライズレベルの可用性、セキュリティ、そしてパフォーマンスがご利用できます。
 
-
-As crazy as it sounds, we are not the first to do this. You can experience this with other serverless services like DynamoDB, S3, and more—we are just bringing serverless to caching!
+これは狂気じみたように聞こえるかもしれませんが、私たちが最初に始めたわけではありません。ほかのサーバーレスサービス、例えばDynamoDB、S3、そして他にもたくさんのサービスで同じものを経験できるでしょう。私たちは単にサーバーレスのアイデアをキャッシュにもってきただけなのです。
 </details>
 
 <details>
-  <summary>Will it scale?</summary>
-Hell yes! Momento is the best way to future-proof your caching story. You can add Momento *extremely* quickly—and it just fades into the background whether you are doing 1 RPS or 1 million RPS. You just pay for what you use—so you get a scalable cache without spending a lot of cash.
+  <summary>スケールするのですか？</summary>
+はい、もちろんです！Momento は将来も使い続けられるキャッシュとして最高の選択肢です。Momento を追加するのは *ものすごく* 素早くできて、そのあとは1RPS だろうが100万RPS だろうが、あとのことは全てバックグラウンドに行われるので皆さんが考える必要がありません。使った分だけ料金を払うだけでよいので、スケール可能なキャッシュを高額な料金を支払うことなく得られます。
 </details>
   
-<details>
-  <summary>Will it blend?</summary>
-We do not recommend putting Momento in a blender as it may void the warranty of your blender, but Momento is robust with enterprise grade security and availability. Momento does blend seamlessly with your current cloud setup, whether you're on AWS, GCP, Azure, or multi-cloud!
-</details>
-
-[Give Momento a Try!](getting-started.md)
+[Momento をはじめてみましょう！](getting-started.md)
```

Comment:
Not sure why but the link is not displayed properly...

<img width="831" alt="image" src="https://user-images.githubusercontent.com/22686260/204641334-2e7a88ad-c9ab-4c14-a75c-9be6ce6bc119.png">

Reply:
Sorry, this was caused by unnecessary whitespaces on the previous line. Fixed by the new commit.